### DISCUSSION TITLE
Unable to redirect URL using http.send in opa policy

### DISCUSSION THREAD

**Discussion by bvamshidhar** [open-policy-agent/community#591] [Community]
Environment is Kubernetes

#rego policy

package example

    import data.http
    import input

    default allow = true

    allow {
        allow_decision := true
        input.method == "GET"
        response := http.send({
            "method": "GET",
            "url": "<url>",
            "body": {
                "allow": allow_decision
            },
            "headers":{
                "Content-Type": "application/json"
            }
        })
        response.status_code == 200
    }

I get this error:
error: initialization error: 1 error occurred: /policy/policy.rego:12: rego_type_error: undefined function data.http.send

**Comment by srenatus** [comment]
Please try removing
```
import data.http
```
It's not needed to use a built-in and actually messes things up here.

**Reply by anderseknert** [reply]
Indeed. Using the [Regal](https://github.com/styrainc/regal) linter would help you catch mistakes like this, and each linter rule comes with documentation to help you understand why a rule exists, and what you can do instead.

You can also see the linter issues reported in the Rego Playground: https://play.openpolicyagent.org/p/yYGBt0zj4G
